cms/templates/js/course-outline.underscore

```diff
@@ -177,6 +177,17 @@ if (is_proctored_exam) {
                         <% } %>
                     </p>
                 </div>
+                <div class="status-hide-after-due">
+                    <p>
+                        <% if (xblockInfo.get('hide_after_due')) { %>
+                            <i class="icon fa fa-eye-slash"></i>
+                            <span class="status-hide-after-due-value"> <%= gettext("Exam will remain hidden after due date") %> </span>
+                        <% } else { %>
+                            <i class="icon fa fa-eye"></i>
+                            <span class="status-hide-after-due-value"> <%= gettext("Exam will be visible after due date") %> </span>
```

Comment:
@chris-mike Do you like always having a message visible? I could also see only including this visibility piece of information in the case of hide_after_due being True (the non-default case).

FYI @catong these changes will need to be documented.